pypesto/select/method.py

```diff
@@ -0,0 +1,462 @@
+import logging
```

Comment:
at top of each file, a `"""One-line description of file contents."""` would be useful.